driver/modern_bpf/helpers/extract/extract_from_kernel.h

```diff
@@ -53,6 +55,25 @@ static __always_inline u32 extract__syscall_id(struct pt_regs *regs)
 #endif
 }
 
+static __always_inline bool extract__32bit_syscall()
```

Comment:
The name is a bit misleading here. I would prefer `bpf_in_ia32_syscall` similar like in the bpf driver. However, I understand that almost all helpers use the `extract__`prefix.

Reply:
Yep that's the reason why i named it like that :) I can change it though.